src/elastic/client.ts

```diff
@@ -0,0 +1,40 @@
+import { Client } from '@elastic/elasticsearch'
+import config from 'config'
+
+export interface ElasticConfig{
+  cloudID: string
+  username: string
+  password: string
+  apiKey: string
+}
+
+const {
+  cloudID,
+  username,
+  password,
+  apiKey
+} = config.get<ElasticConfig>('elastic')
+
+const devAuth = {
+  username: username,
+  password: password,
+}
+
+const prodAuth = {
+  apiKey
+}
+
+const auth = (() => {
+  if(process.env.NODE_ENV === 'development') {
+    console.log('Using development elastic auth credentials')
+    return devAuth
+  }
```

Comment:
In dev we're getting unrestricted access so we can do things like seeding the database. In prod we auth with an apiKey that only lets us read and write specific things